javascript/react/intro-react/what-is-jsx.md

```diff
@@ -22,41 +22,59 @@ parent: what-is-react
 
 ---
 
-# What is **JSX**?
+# What is JSX?
 
 ---
 ## Content
 
-**JSX** is syntactic sugar that brings an `XML`-like syntax to **JavaScript**.
+**JSX** is syntactic sugar that brings an `XML`-like syntax to **JavaScript**. Here is how it looks like:
 
+```jsx
+const el = <h2>Enki!</h2>;
+```
 
-This means that **JSX** is a way of writing your **HTML** inside `JS` code.
-
+This means that **JSX** is a way of writing your **HTML** inside JS code.
 
 A **JSX** expression is evaluated to a **React** `element`, which is short-hand for calling `React.createElement()` Think of `ReactElement`s as the **UI** building blocks of **React**. We will come back to this later.
 
-**JSX** expressions are transformed (compiled) into standard `JS` objects.
+**JSX** expressions are transformed (compiled) into standard JS objects. For example, the following **JSX** code:
 
-For example, the following **JSX** code:
 ```jsx
 const el = <h2>Enki!</h2>;
 ```
+
 will be *compiled* to:
-```
-const el = React.createElement('h2',
- null, 'Enki!');
+
+```jsx
+const el = React.createElement(
+  "h2",
+  null,
+  "Enki!"
+);
 ```
 
-From here, you can embed `JS` expressions within **JSX** by wrapping them with `{ }` curly brackets:
+After performing some checks that ensure your code is bug-free, the `React.createElement()` method creates an object like this:
+
+```jsx
+const el = {
+  type: 'h2',
+  props: {
+    children: 'Enki!'
+  }
+};
 ```
+
+From here, you can embed JS expressions within **JSX** by wrapping them with `{ }` curly brackets:
+
+```jsx
 const x = "Hey there!";
 
-const el = <h2>{x}</h2>
+const el = <h2>{x}</h2>;
 ```
 
-When rendered, this element will display `"Hey there"` as a `h2`.
+When rendered, this element will display `"Hey there"` as an `h2`.
 
-**JSX** expressions can be used everywhere in `JS`. You can:
+**JSX** expressions can be used everywhere in JS. You can:
```

Comment:
```suggestion
Because we are using Babel to transform our entire React application, you can us **JSX** expressions anywhere. For example, you can:
```